src/notifier/index.ts

```diff
@@ -1,12 +1,26 @@
 import { getAllExpiringDocs } from "../sheets/api";
-import { DAYS_TO_VISA_EXPIRY, DAYS_TO_REGISTRATION_EXPIRY } from "../db/utils";
+import {
+  DAYS_TO_VISA_EXPIRY,
+  DAYS_TO_REGISTRATION_EXPIRY,
+  DAYS_TO_REGISTRATION_EXPIRY_FOR_TEMP_RES,
+} from "../db/utils";
 import { sequelize, User } from "../db";
-import { expiringRegistrationMessage, expiringVisaMessage, katyaNotification } from "./messages";
+import {
+  expiringRegistrationMessage,
+  expiringVisaMessage,
+  expiringRegistrationForTemporaryResidency,
+  katyaNotification,
+} from "./messages";
 import { User as SheetUser } from "../sheets/user";
 import { Op } from "sequelize";
 
-async function getStudentsForDoc(users: SheetUser[], type: "visa" | "registration") {
-  const days = type === "visa" ? DAYS_TO_VISA_EXPIRY : DAYS_TO_REGISTRATION_EXPIRY;
+async function getStudentsForDoc(users: SheetUser[], type: "visa" | "registration" | "residency") {
+  const days =
+    type === "visa"
+      ? DAYS_TO_VISA_EXPIRY
+      : type === "registration"
+      ? DAYS_TO_REGISTRATION_EXPIRY
+      : DAYS_TO_REGISTRATION_EXPIRY_FOR_TEMP_RES;
```

Comment:
Since the types are increasing, it's better to use an object for mapping as such:
```ts
type Doc = 'visa' | 'registration' | 'residency';
const docThresholds: Record<Doc, number> = {
  visa: DAYS_TO_VISA_EXPIRY,
  // ... the rest
};
const days = docThresholds[type];